src/Core/Managed/Net40/Extensibility/Implementation/TelemetryConfigurationFactory.cs

```diff
@@ -185,6 +214,20 @@ protected static void LoadProperties(XElement instanceDefinition, object instanc
                             property.SetValue(instance, propertyValue, null);
                         }
                     }
+                    else if (propertyName == "TelemetryProcessors")
+                    {
+                        if (properties.TryGetValue("TelemetryProcessor", out property))
+                        {                            
+                            ITelemetryProcessor currentTP = (ITelemetryProcessor)property.GetValue(instance, null);
+
+                            ITelemetryProcessor newTP = LoadInstancesTelemetryProcessors(propertyDefinition, currentTP);
+                            
+                            if (property.CanWrite && newTP != null)
```

Comment:
the division of responsibility of initialization seems too spread here across of TelemetryConfiguration, LoadInstances<T> and LoadInstancesTelemetryProcessors...see other comment for details...